Documentation/ExtensionArchitecture/Extbase/Introduction/Index.rst

```diff
@@ -0,0 +1,92 @@
+.. include:: /Includes.rst.txt
+.. index::
+   pair: Extension development; Extbase introduction
+
+.. _extbase-introduction:
+
+====================
+Extbase introduction
+====================
+
+.. todo:: Refer to a good example, e.g. tea, blog_example?
+
+.. todo:: Add tips how to get more information or help
+
+What is Extbase?
+================
+
+Extbase is provided via a TYPO3 system extension (typo3-cms/extbase). Extbase is
+a framework within the TYPO3 Core. The framework is based on the software pattern
+`MVC (model-view-controller) <https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller>`__
+and uses
+`ORM (object relational modeling) <https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping>`__.
+
+Extbase can be and is often used in combination with the Fluid templating engine,
+but Fluid **can** also be used without Extbase. All things that can be done with
+Extbase, can also be done with TYPO3 Core native functionality.
+Extbase is not a prerequisite for extension development. In most cases,
+using Extbase means writing less code, but the performance may suffer.
+
+When Extbase was released, it was introduced as the modern way to program
+extensions and the "old" way (pibase) was propagated as outdated. When we look
+at this today, it is not entirely true: Extbase is a good fit for some specific
+types of extensions and there are always alternatives. For some use cases it
+is not a good fit at all and the extension can and should be developed without
+Extbase.
+
+Thus, many things, such as backend modules or plugins can be done "the Extbase
+way" or "the Core way". This is a design decision, the extension developer must
+make for the specific use case.
+
+.. important::
+
+   Extbase is one of many options in the "toolbox" of TYPO3 extension development.
+
+   TYPO3 is flexible, often provides several approaches for one thing and it is
+   up to the extension developer to make an informed choice.
+
+
+Extbase or not?
+===============
+
+When to use Extbase and when to use other methods?
+
+As a rule of thumb (which should not be blindly followed but gives some
+guidance):
+
+Use Extbase if you:
+
+*  wish to get started quickly, e.g. using the
+   :ref:`extension_builder <extension-builder>`
+*  are a beginner or do not have much experience with TYPO3
+*  want to create a "classic" Extbase extension with plugins and (possibly)
+   backend modules (as created with the extension_builder)
```

Comment:
Explain what the extension_builder is and link to more info.

Reply:
We are working on explaining the extension builder better. However the information is not ready yet. Will link it as soon as its available